packages/structured-clone/build.js

```diff
@@ -27,6 +27,7 @@ import fs from "node:fs/promises";
     esbuild
         .build({
             ...commonConfig,
+            external: commonConfig.external.filter(dep => !/unified|bail|is-plain-obj|trough|vfile.*|unist.*|hast.*|property-information|html-void-elements|.*-separated-tokens|.*entities.*|ccount|rehype*|string-width|strip-ansi|ansi-regex|supports-color|rehype|web-namespaces|zwitch/.exec(dep)),
```

Comment:
Can we put this regex in a common file and import it into the build scripts? That way it'll be easier to add items in the future.